projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -64,6 +84,18 @@ export class TabParagraphContainerComponent
     this.activeTabNum = tabNum;
   }
 
+  ngOnInit(): void {
+    if (this.winRef && this.winRef.nativeWindow) {
+      const routeState =
```

Comment:
Actually I didn't read your message properly sorry. What you should do here to read it is: `this.router.getCurrentNavigation().extras.state` So getting it from the Router. Not sure if you can get it from the Routing Service

Reply:
I cannot get it from `RoutingService`. And when this component is dynamically created, the navigation is done. So, `this.router.getCurrentNavigation()` always return `null`, even I put it in `constructor`.